runelite-client/src/test/java/net/runelite/client/plugins/raidsthieving/RaidsTheivingTest.java

```diff
@@ -0,0 +1,208 @@
+/*
+ * Copyright (c) 2018, Tim Lehner <Timothy.Lehner.2011@live.rhul.ac.uk>
+ * All rights reserved.
+ *
+ * Redistribution and use in source and binary forms, with or without
+ * modification, are permitted provided that the following conditions are met:
+ *
+ * 1. Redistributions of source code must retain the above copyright notice, this
+ *    list of conditions and the following disclaimer.
+ * 2. Redistributions in binary form must reproduce the above copyright notice,
+ *    this list of conditions and the following disclaimer in the documentation
+ *    and/or other materials provided with the distribution.
+ *
+ * THIS SOFTWARE IS PROVIDED BY THE COPYRIGHT HOLDERS AND CONTRIBUTORS "AS IS" AND
+ * ANY EXPRESS OR IMPLIED WARRANTIES, INCLUDING, BUT NOT LIMITED TO, THE IMPLIED
+ * WARRANTIES OF MERCHANTABILITY AND FITNESS FOR A PARTICULAR PURPOSE ARE
+ * DISCLAIMED. IN NO EVENT SHALL THE COPYRIGHT OWNER OR CONTRIBUTORS BE LIABLE FOR
+ * ANY DIRECT, INDIRECT, INCIDENTAL, SPECIAL, EXEMPLARY, OR CONSEQUENTIAL DAMAGES
+ * (INCLUDING, BUT NOT LIMITED TO, PROCUREMENT OF SUBSTITUTE GOODS OR SERVICES;
+ * LOSS OF USE, DATA, OR PROFITS; OR BUSINESS INTERRUPTION) HOWEVER CAUSED AND
+ * ON ANY THEORY OF LIABILITY, WHETHER IN CONTRACT, STRICT LIABILITY, OR TORT
+ * (INCLUDING NEGLIGENCE OR OTHERWISE) ARISING IN ANY WAY OUT OF THE USE OF THIS
+ * SOFTWARE, EVEN IF ADVISED OF THE POSSIBILITY OF SUCH DAMAGE.
+ */
+package net.runelite.client.plugins.raidsthieving;
+
+import net.runelite.client.plugins.raidsthieving.BatSolver.BatSolver;
+import net.runelite.client.plugins.raidsthieving.BatSolver.ThievingRoomType;
+import org.junit.Test;
+import org.junit.runner.RunWith;
+import org.mockito.runners.MockitoJUnitRunner;
+
+import java.util.Arrays;
+import java.util.TreeSet;
+
+import static net.runelite.client.plugins.raidsthieving.BatSolver.Matrix2D.MatrixMultiple2D;
+import static net.runelite.client.plugins.raidsthieving.BatSolver.Matrix2D.Rotate90;
+
+@RunWith(MockitoJUnitRunner.class)
+public class RaidsTheivingTest
+{
+	private boolean compare(TreeSet<Integer> set1, TreeSet<Integer> set2) {
+		return set1.containsAll(set2) && set2.containsAll(set1);
+	}
+
+	@Test
+	public void testBatFinderCHECK_GRUBS_INVALIDATES_SOLN()
+	{
+		BatSolver solver = new BatSolver(ThievingRoomType.LEFT_TURN);
+
+		solver.addEmptyChest(1);
+
+		TreeSet<Integer> compareSet1 = new TreeSet<>(Arrays.asList(1, 16, 17, 38, 54, 55));
+		TreeSet<Integer> compareSet2 = new TreeSet<>(Arrays.asList(1, 17, 38, 54));
+
+		assert compare(compareSet1, solver.matchSolutions());
```

Comment:
Use JUnit's Assert class in tests so they don't depend on the vm's assertion flag

Reply:
Fixed